common/protob/messages-cardano.proto

```diff
@@ -38,6 +38,15 @@ enum CardanoPoolRelayType {
     MULTIPLE_HOST_NAME = 2;
 }
 
+enum CardanoTxAuxiliaryDataSupplementType {
+    CATALYST_REGISTRATION_SIGNATURE = 0;
+}
+
+enum CardanoTxSigningMode {
+    ORDINARY_TRANSACTION = 0;
+    POOL_REGISTRATION = 1;
```

Comment:
this should perhaps be POOL_REGISTRATION_AS_OWNER - on Ledger the policies for owners/operators differ, do I'd keep it consistent and also make it obvious that only owners are supported in Trezor (right now). If we happen to be able to have one policy, we can rename the enum after all